app/src/main/java/com/duckduckgo/app/notification/model/SearchPromptNotification.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.notification.model
+
+import android.content.Context
+import androidx.core.app.NotificationManagerCompat
+import com.duckduckgo.app.browser.R
+import com.duckduckgo.app.notification.NotificationHandlerService
+import com.duckduckgo.app.notification.NotificationRegistrar
+import com.duckduckgo.app.notification.db.NotificationDao
+import com.duckduckgo.app.settings.db.SettingsDataStore
+import com.duckduckgo.app.statistics.VariantManager
+import timber.log.Timber
+
+class SearchPromptNotification(
+    private val context: Context,
+    private val notificationDao: NotificationDao,
+    private val variantManager: VariantManager,
+    private val settingsDataStore: SettingsDataStore
+) : SearchNotification {
+
+    override val id = "com.duckduckgo.privacy.search.stickyPrompt"
+
+    override val pressIntent = NotificationHandlerService.NotificationEvent.QUICK_SEARCH_PROMPT_LAUNCH
+
+    override val launchIntent = NotificationHandlerService.NotificationEvent.QUICK_SEARCH_KEEP
+
+    override val cancelIntent = NotificationHandlerService.NotificationEvent.QUICK_SEARCH_REMOVE
+
+    override val layoutId = R.layout.search_notification_prompt
+
+    override val priority = NotificationManagerCompat.IMPORTANCE_DEFAULT
+
+    override suspend fun canShow(): Boolean {
```

Comment:
Threre's some logic not covered by test inside this method. I would cover the database logic at least, what do you think?

Reply:
I followed the same approach as per the other notifications already created as they don't test it. You're probably right, but I think that should be a separate task to take care in LHF days. `Improve test coverage in Notifications` or something along those lines.